recipes/cotire/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "1.7.8" %}
+{% set sha256 = "84be4ff66248ba629666bed5c2b76b869db7f04425e44bba3555bbc25d5701c1" %}
+
+package:
+  name: cotire
+  version: {{ version }}
+
+source:
+  fn: cotire-{{ version }}.tar.gz
+  url: https://github.com/sakra/cotire/archive/cotire-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Include cotire in example CMakeLists.txt.
+    #
+    # https://github.com/sakra/cotire/pull/100
+    #
+    - PR_100.diff
+
+build:
+  number: 0
+  script:
+    - cp "${SRC_DIR}/CMake/cotire.cmake" "${PREFIX}"/share/cmake*/Modules               # [unix]
+    - copy "%SRC_DIR%\\CMake\\cotire.cmake" "%LIBRARY_PREFIX%"\\share\\cmake*\\Modules  # [win]
```

Comment:
Am I doing something wrong here, @msarahan? It seem to have issues copying this on Windows. I really don't know the batch syntax.